dev/dags/example_virtualenv.py

```diff
@@ -56,6 +56,16 @@ def example_virtualenv() -> None:
             "py_requirements": ["dbt-postgres"],
             "install_deps": True,
             "emit_datasets": False,  # Example of how to not set inlets and outlets
+            # --------------------------------------------------------------------------
```

Comment:
If you could explain why these are commented out, it would be great!

Reply:
added a comment to explain it